opendemic/human/model.py

```diff
@@ -383,10 +383,21 @@ def create_human(telegram_human_id: int = None, fingerprint: str = None) -> Tupl
 				mysql_db_format_value(value=fingerprint)
 			)
 		)
-	except Exception as e:
-		logger.error(e)
-		err = e
-	return Human(human_id=human_id), err
+	except Exception as query_creation_err:
```

Comment:
We should not be checking for both the exception and the `db_err`. In an exception is thrown in the query, it will be part of `db_err. So I'd remove the except part and just check for db_err

Reply:
Not quite. The db_err would be thrown within the rdb.execute method. But there is still a possibility that an exception be thrown when the query template is being populated with values. This has happened a few times in the past. Hence why they are separate.

Reply:
Oh I see, then it makes sense